From a5e6642411deee48a10c3bf92d91fac63c2c61bb Mon Sep 17 00:00:00 2001
From: Alberto Ponces <ponces26@gmail.com>
Date: Wed, 25 Dec 2019 12:34:05 +0200
Subject: [PATCH] Rework securize tweak

---
 rw-system.sh | 38 +++++++++++++++++++++++++++++++-------
 1 file changed, 31 insertions(+), 7 deletions(-)

diff --git a/rw-system.sh b/rw-system.sh
index 5dc493f..116ae14 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -739,35 +739,59 @@ copyprop() {
         resetprop_phh "$1" "$(getprop "$2")"
     fi
 }
-if [ -f /system/phh/secure ] || [ -f /metadata/phh/secure ];then
     copyprop ro.build.device ro.vendor.build.device
     copyprop ro.system.build.fingerprint ro.vendor.build.fingerprint
     copyprop ro.bootimage.build.fingerprint ro.vendor.build.fingerprint
     copyprop ro.build.fingerprint ro.vendor.build.fingerprint
+    copyprop ro.system_ext.build.fingerprint ro.vendor.build.fingerprint
+    copyprop ro.product.build.fingerprint ro.vendor.build.fingerprint
     copyprop ro.build.device ro.vendor.product.device
     copyprop ro.product.system.device ro.vendor.product.device
+    copyprop ro.yaap.device ro.product.vendor.device
     copyprop ro.product.device ro.vendor.product.device
     copyprop ro.product.system.device ro.product.vendor.device
     copyprop ro.product.device ro.product.vendor.device
+    copyprop ro.product.system_ext.device ro.vendor.product.device
+    copyprop ro.product.product.device ro.vendor.product.device
+    copyprop ro.product.system_ext.device ro.product.vendor.device
+    copyprop ro.product.product.device ro.product.vendor.device
     copyprop ro.product.system.name ro.vendor.product.name
     copyprop ro.product.name ro.vendor.product.name
-    copyprop ro.product.system.name ro.product.vendor.device
-    copyprop ro.product.name ro.product.vendor.device
-    copyprop ro.system.product.brand ro.vendor.product.brand
+    copyprop ro.product.system.name ro.product.vendor.name
+    copyprop ro.product.name ro.product.vendor.name
+    copyprop ro.product.system_ext.name ro.vendor.product.name
+    copyprop ro.product.product.name ro.vendor.product.name
+    copyprop ro.product.system_ext.name ro.product.vendor.name
+    copyprop ro.product.product.name ro.product.vendor.name
     copyprop ro.product.brand ro.vendor.product.brand
+    copyprop ro.product.system.brand ro.vendor.product.brand
+    copyprop ro.product.system_ext.brand ro.vendor.product.brand
+    copyprop ro.product.product.brand ro.product.vendor.brand
+    copyprop ro.system.product.brand ro.product.vendor.brand
+    copyprop ro.product.brand ro.product.vendor.brand
+    copyprop ro.product.system.brand ro.product.vendor.brand
+    copyprop ro.product.system_ext.brand ro.product.vendor.brand
+    copyprop ro.product.product.brand ro.product.vendor.brand
     copyprop ro.product.system.model ro.vendor.product.model
     copyprop ro.product.model ro.vendor.product.model
+    copyprop ro.product.system_ext.model ro.vendor.product.model
+    copyprop ro.product.product.model ro.vendor.product.model
     copyprop ro.product.system.model ro.product.vendor.model
     copyprop ro.product.model ro.product.vendor.model
     copyprop ro.build.product ro.vendor.product.model
     copyprop ro.build.product ro.product.vendor.model
+    copyprop ro.product.system_ext.model ro.product.vendor.model
+    copyprop ro.product.product.model ro.product.vendor.model
     copyprop ro.system.product.manufacturer ro.vendor.product.manufacturer
     copyprop ro.product.manufacturer ro.vendor.product.manufacturer
+    copyprop ro.product.system.manufacturer ro.vendor.product.manufacturer
+    copyprop ro.product.product.manufacturer ro.vendor.product.manufacturer
+    copyprop ro.product.system_ext.manufacturer ro.vendor.product.manufacturer
     copyprop ro.system.product.manufacturer ro.product.vendor.manufacturer
     copyprop ro.product.manufacturer ro.product.vendor.manufacturer
-    (getprop ro.vendor.build.security_patch; getprop ro.keymaster.xxx.security_patch) |sort |tail -n 1 |while read v;do
-        [ -n "$v" ] && resetprop_phh ro.build.version.security_patch "$v"
-    done
+    copyprop ro.product.system.manufacturer ro.product.vendor.manufacturer
+    copyprop ro.product.product.manufacturer ro.product.vendor.manufacturer
+    copyprop ro.product.system_ext.manufacturer ro.product.vendor.manufacturer
 
     resetprop_phh ro.build.tags release-keys
     resetprop_phh ro.boot.vbmeta.device_state locked
-- 
2.39.2

