From d328121fd4ecb0d81de3d10336cf83b7643b1fb3 Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Fri, 23 Jun 2023 18:26:57 +0500
Subject: [PATCH] Make Builds read only (to reduce file sizes).

---
 board-base.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board-base.mk b/board-base.mk
index c38e385..97a4d02 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -11,3 +11,5 @@ BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 BUILD_BROKEN_DUP_RULES := true
 
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
+
+BOARD_EXT4_SHARE_DUP_BLOCKS := true
-- 
2.40.1

